bin/wasmtime-http.rs

```diff
@@ -51,8 +56,13 @@ async fn main() -> Result<(), Error> {
     let opt = Opt::from_args();
     let method = opt.invoke.clone();
     // println!("{:?}", opt);
```

Comment:
Can we remove this or wrap it in debug block if required for that purpose?

Reply:
Good point.
Since it's not related to this changeset, I'll address it in a future PR.